core/src/test/java/org/elasticsearch/action/ActionModuleTests.java

```diff
@@ -99,32 +109,69 @@ public ActionResponse newResponse() {
     }
 
     public void testSetupRestHandlerContainsKnownBuiltin() {
-        assertThat(ActionModule.setupRestHandlers(emptyList()), hasItem(RestMainAction.class));
+        SettingsModule settings = new SettingsModule(Settings.EMPTY);
```

Comment:
I switched these to assert based on paths instead of class which makes more sense in the new world.